src/rustbox.rs

```diff
@@ -89,22 +88,43 @@ mod style {
 
 const NIL_RAW_EVENT: RawEvent = RawEvent { etype: 0, emod: 0, key: 0, ch: 0, w: 0, h: 0, x: 0, y: 0 };
 
-// FIXME: Rust doesn't support this enum representation.
-// #[derive(Copy,FromPrimitive,Debug)]
-// #[repr(C,int)]
-// pub enum EventErrorKind {
-//     Error = -1,
-// }
-// pub type EventError = Option<EventErrorKind>;
-#[allow(non_snake_case)]
-pub mod EventErrorKind {
-    #[derive(Clone, Copy,Debug)]
-    pub struct Error;
+#[derive(Debug)]
+pub enum EventError {
```

Comment:
I don't know if this is even needed because there is no "type" of error for Termbox. It just says it returns -1 if there was an error. Could just wrap `isize` type:
```rust
pub struct EventError(isize);
```